Directory.Packages.Props

```diff
@@ -0,0 +1,103 @@
+<Project>
+	<PropertyGroup>
+		<ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+		<CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
+		<!-- Used many times below-->
+		<MicrosoftVisualStudioPackageCurrent>17.10.40119</MicrosoftVisualStudioPackageCurrent>
+	</PropertyGroup>
+	<PropertyGroup>
+		<EntityFrameworkPackageVersion>6.5.1</EntityFrameworkPackageVersion>
```

Comment:
Not all properties removed in `Directory.Build.props` have made it through to here. Just want to make sure that is intentional. (guess it wouldn't matter as long as the projects build correctly with the new setup)

Reply:
There are two variables that were present in Directory.Build.props that aren't in Directory.Packages.props.  The first is MySqlDataEntityPackageVersion, for which there is no longer any reference to it.  The second is for NerdBank.Streams, which is presently inlined.  I think we're fine here.